nvm.sh

```diff
@@ -1545,6 +1568,13 @@ nvm() {
       if [ "_$1" = "_-s" ]; then
         nobinary=1
         shift
+        if [ "_$1" = "_-j" ]; then
```

Comment:
`-j -s` should probably error out, as should `-j $not_a_number` - i think it should be checked along with `-s`, and ideally in any order.